.github/workflows/java-microservice-ci-default.yml

```diff
@@ -188,6 +188,13 @@ jobs:
           name: cyclonedx-bom-xml
           path:  ${{github.workspace}}/target/bom.xml
 
+      - name: Upload test reports
+        uses: actions/upload-artifact@v4
+        if: ${{ always() }}
```

Comment:
If the process fail before tests what happens ?

Reply:
No, if no files find, this steps do nothing